tutorials/katacoda/basic-pipeline/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
no need to include the LICENSE file with each scenario as it's already included in the project root

Reply:
removed :)